taskcluster/scripts/toolchain/android-gradle-dependencies.sh

```diff
@@ -19,15 +19,15 @@ pushd $PROJECT_DIR
 . taskcluster/scripts/toolchain/android-gradle-dependencies/before.sh
 
 COMPONENT_REGEX='^  ([-a-z]+):$'
-# XXX We don't build samples-browsers at all. They don't have any extra dependencies so we can
-# filter them out. Other components are built in second pass
-COMPONENTS_TO_EXCLUDE_IN_FIRST_PASS='(samples-browser|tooling-detekt|tooling-lint)'
+# Components to ignore on first pass. They don't have any extra dependencies so we can
+# filter them out. Other components are built in second pass.
+COMPONENTS_TO_EXCLUDE_IN_FIRST_PASS='(tooling-detekt|tooling-lint)'
 FIRST_PASS_COMPONENTS=$(grep -E "$COMPONENT_REGEX" "$PROJECT_DIR/.buildconfig.yml" | sed -E "s/$COMPONENT_REGEX/:\1/g" | grep -E -v "$COMPONENTS_TO_EXCLUDE_IN_FIRST_PASS")
 
 ASSEMBLE_COMMANDS=$(echo "$FIRST_PASS_COMPONENTS" | sed "s/$/:assemble/g")
 ASSEMBLE_TEST_COMMANDS=$(echo "$FIRST_PASS_COMPONENTS" | sed "s/$/:assembleAndroidTest/g")
 TEST_COMMANDS=$(echo "$FIRST_PASS_COMPONENTS" | sed "s/$/:test/g")
-LINT_COMMANDS=$(echo "$FIRST_PASS_COMPONENTS" | sed "s/$/:lintRelease/g")
```

Comment:
There's no `lintRelease` in Sample Browser, so switching to `lint`.

Reply:
I don't remember why we prefer using `lintRelease` over `lint`. That said, if we change this value here, we need to update https://github.com/mozilla-mobile/android-components/blob/f074469a2fe08cf0ca52e57d86d45c2900d9ff88/taskcluster/ci/build/kind.yml#L86. 

In my opinion, we should keep filtering out `samples-browser` in the first pass, and add it alongside `tooling-lint`https://github.com/mozilla-mobile/android-components/blob/f074469a2fe08cf0ca52e57d86d45c2900d9ff88/taskcluster/scripts/toolchain/android-gradle-dependencies.sh#L43-L46

Both have no `lintRelease` per https://github.com/mozilla-mobile/android-components/blob/f074469a2fe08cf0ca52e57d86d45c2900d9ff88/taskcluster/ci/build/kind.yml#L70

What do you think @csadilek?